chain/epoch-manager/src/lib.rs

```diff
@@ -713,26 +715,28 @@ impl EpochManager {
         ) {
             Ok(next_next_epoch_info) => next_next_epoch_info,
             Err(EpochError::ThresholdError { stake_sum, num_seats }) => {
-                warn!(target: "epoch_manager", "Not enough stake for required number of seats (all validators tried to unstake?): amount = {} for {}", stake_sum, num_seats);
+                tracing::warn!(target: "epoch_manager", %stake_sum, %num_seats, "not enough stake for required number of seats (all validators tried to unstake?)");
                 let mut epoch_info = EpochInfo::clone(&next_epoch_info);
                 *epoch_info.epoch_height_mut() += 1;
                 epoch_info
             }
             Err(EpochError::NotEnoughValidators { num_validators, num_shards }) => {
-                warn!(target: "epoch_manager", "Not enough validators for required number of shards (all validators tried to unstake?): num_validators={} num_shards={}", num_validators, num_shards);
+                tracing::warn!(target: "epoch_manager", %num_validators, %num_shards, "not enough validators for required number of shards (all validators tried to unstake?)");
                 let mut epoch_info = EpochInfo::clone(&next_epoch_info);
                 *epoch_info.epoch_height_mut() += 1;
                 epoch_info
             }
             Err(err) => return Err(err),
         };
         let next_next_epoch_id = EpochId(*last_block_hash);
-        debug!(target: "epoch_manager", "next next epoch height: {}, id: {:?}, protocol version: {} shard layout: {:?} config: {:?}",
-               next_next_epoch_info.epoch_height(),
-               &next_next_epoch_id,
-               next_next_epoch_info.protocol_version(),
-               self.config.for_protocol_version(next_next_epoch_info.protocol_version()).shard_layout,
-            self.config.for_protocol_version(next_next_epoch_info.protocol_version()));
+        tracing::debug!(target: "epoch_manager",
+            epoch_height = %next_next_epoch_info.epoch_height(),
+            ?next_next_epoch_id,
+            protocol_version = %next_next_epoch_info.protocol_version(),
+            shard_layout = ?self.config.for_protocol_version(next_next_epoch_info.protocol_version()).shard_layout,
+            config = ?self.config.for_protocol_version(next_next_epoch_info.protocol_version()),
+            "next next epoch"
+        );
```

Comment:
Could you please adjust names to align with previous version and remove message?